src/style.css

```diff
@@ -1 +1,111 @@
-/* add styles here */
+* {
```

Comment:
The use of the universal selector `*` can impact performance. While not critical, it's a good practice to avoid it for better performance. Consider using more specific selectors.